InfoLogger/lib/utils/preparedStatementParser.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+/**
+ * Translate the SQL prepared statement to a regular SQL query.
```

Comment:
Hi, one last thing. Could you add a comment in this doc block to specifically say that this function should be used for `logging` purposes only